packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import { Image } from 'react-native';
+import { Formik } from 'formik';
+import {
+  ModalView,
+  Switch,
+  TextInput,
+  Button,
+  styled,
+  FlexedView,
+  PaddedView,
+} from '@apollosproject/ui-kit';
+import AddPrayerHeader from '../AddPrayerHeader';
+
+const StyledButton = styled(() => ({
+  borderRadius: 0,
+}))(Button);
+
+const BottomView = styled(() => ({
+  justifyContent: 'flex-end',
+}))(FlexedView);
+
+const AddPrayerForm = memo(({ imgSrc, btnLabel, ...otherProps }) => (
+  <Formik
+    initialValues={{ prayer: '', anonymous: false }}
+    onSubmit={(values) => console.log(values)}
+  >
+    {({ handleChange, handleBlur, handleSubmit, values }) => (
+      <ModalView {...otherProps}>
+        <FlexedView>
+          <AddPrayerHeader imgSrc={imgSrc} />
+          {/* TODO: add underline={false} prop to TextInput, 
+        pending https://github.com/ApollosProject/apollos-prototype/issues/629 */}
+          <PaddedView>
+            <TextInput
+              editable
+              placeholder="Start typing your prayer..."
+              onChangeText={handleChange('prayer')}
+              onBlur={handleBlur('prayer')}
+              value={values.prayer}
+            />
+          </PaddedView>
+          <BottomView>
+            <PaddedView>
+              <Switch
```

Comment:
Looking at the mock below, I'd love to see this Anonymous switch look more like that _if it's possible_. If not, ignore.

![Screenshot 2019-04-18 13 39 44](https://user-images.githubusercontent.com/3229463/56380187-7af52c00-61df-11e9-91b4-f2c6606bcab9.png)

Reply:
I could put it a core PR to swap the side the label is on for the `ui-kit/Switch`?

Reply:
I don't think the sides the things are on are that big of a deal. What if you just centered it? Is that possible?

Reply:
![Screen Shot 2019-04-18 at 2 45 13 PM](https://user-images.githubusercontent.com/2659478/56383737-97e22d00-61e8-11e9-91bb-62afd073e3a5.png)
 what do you think?